Description: uima-examples: disable ecore (Eclipse EMF)
 by using "excludes" parameter of maven-compiler-plugin
Author: Damien Raude-Morvan <drazzib@debian.org>
Last-Update: 2010-04-05
Forwared: not-needed
--- a/uimaj-examples/pom.xml
+++ b/uimaj-examples/pom.xml
@@ -82,7 +82,8 @@
 			<artifactId>uimaj-tools</artifactId>
 			<version>${project.parent.version}</version>
 			<scope>compile</scope>
-		</dependency>		
+		</dependency>	
+<!--	
  		<dependency>
 			<groupId>org.eclipse.emf</groupId>
 			<artifactId>common</artifactId>
@@ -101,6 +102,7 @@
 			<version>2.1.0</version>
 			<scope>provided</scope>
 		</dependency>
+-->
 	</dependencies>
 	<build>
 		<finalName>uima-examples</finalName>
@@ -122,8 +124,17 @@
               </configuration>
             </execution>
           </executions>
-        </plugin>       
+        </plugin>
+        <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <configuration>
+               <excludes>
+                       <exclude>**/org/apache/uima/examples/xmi/**.java</exclude>
+               </excludes>
+            </configuration>
+        </plugin>
       </plugins>
     </pluginManagement>
 	</build>	
-</project>
\ No newline at end of file
+</project>
